lbjava-examples/target/classes/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
I was trying to add an empty directory `classes` under `target`. Git can't just add an empty directory.